CHANGELOG.md

```diff
@@ -88,8 +88,13 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 #### Alphabet
 
-* Removed seqan3::char_is_valid_for requirement from seqan3::writable_alphabet and seqan3::writable_constexpr_alphabet
+* Removed seqan3::char_is_valid_for requirement from seqan3::detail::writable_alphabet and
```

Comment:
This change looks like it would belong to a different PR? Or did you just fix this because you touched the file already?

Reply:
No, that is not necessary. Just wanted to make sure, that this was on purpose and not an accident. ;) 
But, this is still shown as a change? (I think because there is a new line that is not in the master branch?)

Reply:
I added `detail::` and needed to wrap the line, as it exceeded our line limit :)